content/docs/user-guide/how-to/share-a-dvc-cache.md

```diff
@@ -87,3 +91,11 @@ If you're using Git, commit the changes to your project's config file (usually
 $ git add .dvc/config
 $ git commit -m "config external/shared DVC cache"
 ```
+
+<admon type="warn">
+
+When using a shared cache, running `dvc gc` in one project may break some links
+in another project. See more info about
+[cleaning a shared cache](/doc/command-reference/gc#cleaning-shared-cache-or-shared-remote).
```

Comment:
```suggestion
Using `dvc gc` with a shared cache may delete data needed
in another project! See more info. about
[cleaning a shared cache](/doc/command-reference/gc#cleaning-shared-cache-or-shared-remote)
safely.
```

Reply:
`dvc gc` doesn't have shared cache (at least it's harder to read it this way) ...` using `dvc gc` in a project(s) with a shared cache ...`